test/FastTests/Client/Includes.cs

```diff
@@ -51,6 +51,51 @@ public void Can_Load_With_Include()
             }
         }
 
+        [Fact]
+        public void Can_Load_With_Include_Using_IIncludeBuilder()
+        {
+            using (var store = GetDocumentStore())
+            {
+                using (var session = store.OpenSession())
+                {
+                    var address = new Address { City = "London", Country = "UK" };
+                    session.Store(address);
+                    session.Store(new User { Name = "Adam", AddressId = session.Advanced.GetDocumentId(address) });
+
+                    session.SaveChanges();
+                }
+
+                using (var session = store.OpenSession())
+                {
+                    var user = session.Load<User>(
+                        "users/1-A",
+                        includes => includes
+                            .Documents(x => x.AddressId));
```

Comment:
lambda usually have shorter names.
How about if we do it like:
```
       var user = session.Load<User>(
                        "users/1-A",
                        i=> i.IncludeDocuments(x => x.AddressId));
```

Reply:
And I'm not sure about `Documents` or `Document`

Reply:
The `Documents` are loaded by a path, so it is hard to say if one or many will be loaded. I would stay with `Documents`.

Reply:
So, IncludeDocuments or just Documents?

Reply:
add `Include` prefix to all methods there.